neuromation/cli/stats.py

```diff
@@ -130,10 +131,15 @@ async def send(client: Client, uid: str, data: List[sqlite3.Row]) -> None:
         )
         + "\n"
     )
+    neuro_ver = neuromation.__version__
+    plat = sys.platform
+    py_version = ".".join(str(i) for i in sys.version_info[:3])
```

Comment:
Maybe use `platform.python_version()`?

And maybe use `platform.python_implementation()` instead of `"Python"`?

Reply:
Thanks. Updated